### PR TITLE
Bats test check file created in previous layer with overlayFS

### PR DESCRIPTION
**Pull request by chofnar**
Signed-off-by: Catalin Hofnar <catalin.hofnar@gmail.com>

### CODE REVIEW

File: test/overlay-dirs.bats [hunk 1]
```diff
@@ -53,6 +53,32 @@ EOF
...
+justbinary:
+    from:
+        type: docker
+        url: docker://gcr.io/distroless/base
+    overlay_dirs:
+        - source: dir_to_overlay
+          dest: /dir_to_overlay
+EOF
+    stacker build
+    ls dir_to_overlay | grep "helloworld.txt"
```
Comment (andaaron) [inline test/overlay-dirs.bats:78]: What we want to verify is the `justbinary` image contains `helloworld.txt`.
Please unpack `justbinary` using skopeo and verify the file is there.

**Review by andaaron — CHANGES_REQUESTED** [review]
Please also add an example with:
```
from:
    type: scratch
```.

Also update this doc with a `scratch` example as an alternative to the other 2.
https://github.com/project-stacker/stacker/blob/master/doc/tricks.md

**Comment by andaaron** [comment]
Updated my previous comment since type;scratch was dropped in https://github.com/project-stacker/stacker/commit/97a80cda7f5b500ad22834ce04a647c219e109c5

File: test/overlay-dirs.bats (hunk 1 above)
Comment (chofnar) [reply test/overlay-dirs.bats:78]: Added, + documentation

**Review by andaaron — APPROVED** [review]

**Pull request merged**